server/mqtt.go

```diff
@@ -1170,6 +1228,20 @@ func (jsa *mqttJSA) deleteStream(name string) (bool, error) {
 	return sdr.Success, sdr.ToError()
 }
 
+func (jsa *mqttJSA) loadLastMsgFor(streamName string, subject string) (*StoredMsg, error) {
+	mreq := &JSApiMsgGetRequest{LastFor: subject}
+	req, err := json.Marshal(mreq)
+	if err != nil {
+		return nil, err
+	}
+	lmri, err := jsa.newRequest(mqttJSAMsgLoad, fmt.Sprintf(JSApiMsgGetT, streamName), 0, req)
```

Comment:
ok to block in place here waiting on response?

Reply:
Yes, all those helpers do that. They are invoked only in places where it is ok to wait.